Source/monster.cpp

```diff
@@ -319,7 +319,8 @@ void PlaceGroup(int mtype, int num, int leaderAttributes, int leaderId)
 		while (placed != 0) {
 			ActiveMonsterCount--;
 			placed--;
-			dMonster[Monsters[ActiveMonsterCount].position.tile.x][Monsters[ActiveMonsterCount].position.tile.y] = 0;
+			const Point &position = Monsters[ActiveMonsterCount].position.tile;

```

Comment:
```suggestion
			const auto &position = Monsters[ActiveMonsterCount].position.tile;
```
Most other places are using `auto` instead of an explicit type for these kinds of aliases.

Reply:
Apparently ALL other places